zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs

```diff
@@ -39,6 +43,22 @@ pub(crate) fn get_num_bits_per_base_chi_lookup() -> usize {
     get_num_bits_per_lookup(CHI_BASE_LOOKUP_RANGE.max(RHO_PI_LOOKUP_RANGE))
 }
 
+pub(crate) trait AssignKeccakRegion {
+    fn assign_keccak_region<F: Field>(&self, region: &mut KeccakRegion<F>, value: F);
+}
+
```

Comment:
This 2 implementation are exactly the same, maybe can move to `trait` default implementation.

https://doc.rust-lang.org/book/ch10-02-traits.html#default-implementations

Reply:
So, the implementation uses self.column_idx and self.rotation. So to do a default implementation I would have to create another trait with get_column_idx() and get_rotation that is implemented by CellValueOnly and Cell.

Do you think I should do that?

Also, if what you says about unifying CellValueOnly and Cell works, perhaps we don't need to do anything of this.

Reply:
You are right, need to introduce getter to have this feature :)

Instead, can seek another option to unify CellValueOnly and Cell 👍